app/src/main/java/fr/free/nrw/commons/nearby/PlaceRenderer.java

```diff
@@ -45,9 +45,11 @@ public void render() {
         tvName.setText(place.name);
         String descriptionText = place.getLongDescription();
         if (descriptionText.equals("?")) {
-            descriptionText = getContext().getString(R.string.no_description_found);
+            tvDesc.setText("");
+        }
+        else{
```

Comment:
Sorry if this is a bit nitpicky, but our style usually utilizes a whitespace after 'else'. :)

Reply:
I agree:)